### PR TITLE
Adds root beer to the game (I FUCKING LOVE MUG!!!!!!!!)

### PR DESCRIPTION
**Pull request by AlvCyktor**
image.
I FUCKING LOVE ROOT BEER I CAN'T LIVE WITHOUT IT AUUUUUUUUUUGH

# Document the changes in your pull request

Adds root-beer to vending machine and the soda dispenser.

# Spriting
Barq
Mug
AW

# Wiki Documentation
Root beer, functions the same as space_cola but have different can sprites.

# Changelog
:cl:  
rscadd: Barq is good apparently 
/:cl:

### CODE REVIEW

**Comment by MommaDragon** [comment]
R O O T
B E E R

**Comment by Spacemanspark** [comment]
ROOT BEER ROOT BEER ROOT BEER ROOT BEER

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -469,6 +469,18 @@
...
+	name = "Root Beer"
+	desc = "A soft drink made from roots. Non-Alcoholic."
+	custom_price = 10
+	icon_state = "Rootbeer_Mug"
+	list_reagents = list(/datum/reagent/consumable/rootbeer = 30)
+	foodtype = SUGAR
+
+/obj/item/reagent_containers/food/drinks/soda_cans/rootbeer/Initialize()
+	icon_state = pick("Rootbeer_Mug","Rootbeer_AW","Rootbeer_Barq")
+	.=..()
```
Comment (tattax) [inline code/modules/food_and_drinks/drinks/drinks.dm]:
```suggestion
	. = ..()
```

**Comment by AlvCyktor** [comment]
thanks tatax

**Comment by tattax** [comment]
it doesn't fix anything it's just that without spaces it looks ugly (and out of line with other . = ..()s)

**Comment by MusselMain** [comment]
As someone who played hockey as a kid, I can confirm this is literally the only thing anyone on our team (including the coach) ever drank after games, and I still unironically love it.

**Comment by Hopekz** [comment]
MUG MUG MUG MUG

**Review by Anvilman6 — APPROVED** [review]

**Comment by Anvilman6** [comment]
Grammar and syntax are fine, as much as I despise putting more memes in the game

**Review by cuackles — APPROVED** [review]
sprites look gud

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**